services/__init__.py

```
@@ -1 +1,34 @@
+    package_name = "services.cad_service"
+    if package_name in sys.modules:
+        return
+
+    package_path = Path(__file__).resolve().parent / "cad-service"
+    spec = importlib.util.spec_from_file_location(
+        package_name,
+        package_path / "__init__.py",
+        submodule_search_locations=[str(package_path)],
+    )
+    if spec is None or spec.loader is None:  # pragma: no cover - defensive
+        raise ImportError("Unable to create module spec for services.cad_service")
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)